CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # Unreleased
 
+# Added
+* Added frame pointer support for subroutine arguments, to substitute excessive use of scratch slots. ([#562](https://github.com/algorand/pyteal/pull/562))
```

Comment:
```suggestion
* Added frame pointer support for subroutine arguments, replacing the previous usage of scratch. ([#562](https://github.com/algorand/pyteal/pull/562))
```
Completely optional rewording that removes judgmental words like "excessive"

Reply:
👍